server/base/pom.xml

```diff
@@ -56,10 +56,20 @@
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.micrometer</groupId>
+      <artifactId>micrometer-core</artifactId>
+      <version>1.6.4</version>
+    </dependency>
     <dependency>
       <groupId>org.apache.accumulo</groupId>
       <artifactId>accumulo-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.accumulo</groupId>
+      <artifactId>accumulo-iterator-test-harness</artifactId>
+      <version>2.1.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
Why include the test-harness? This seems like an undesirable dependency,

Reply:
This must have been added automatically somehow. I'm not sure why.